packages/ui-tree-browser/src/components/TreeBrowser/TreeButton/index.js

```diff
@@ -114,7 +132,33 @@ export default class TreeButton extends Component {
     }
   }
 
-  render () {
+  renderFulltext() {
+    const { name, descriptor } = this.props
+
+    return (
+      <>
```

Comment:
Unfortunately we can't support fragments until we drop support for React 15 (likely in InstUI 7)